src/partner/modules/EventList/EventListItem.tsx

```diff
@@ -2,15 +2,13 @@ import React from "react";
 import { Link } from "react-router-dom";
 import { IEvent } from "src/partner/models/Event";
 import { Modal } from "src/partner/modules/ui/Modal/Modal";
-import { Notification } from "src/partner/modules/ui/Modal/Notification";
 import { CreateEvent } from "src/components/event/Create";
-import styledComponents from "styled-components";
 import { ListStyled } from "src/partner/modules/ui";
-import { NotificationContext } from "src/providers";
 
 export interface IEventItemProps {
   eventInfo: IEvent;
   eventView?: boolean;
+  handleCancelEvent: Function;
```

Comment:
avoid using `Function`, if the function just takes any event a doesn't return anything use:

```ts
export interface IEventListContainerProps {
  handleCancelEvent: (e: any) => void;
}
```

else use a meaningful function signature